spec/moonshot/commands/build_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Moonshot::Commands::Build do
+
+  it 'should not handle --skip-ci-status and build correctly' do
+    cli_dispatcher = Moonshot::CommandLineDispatcher.new('build', subject, {})
+    parser = cli_dispatcher.send(:build_parser, subject)
+    expect { parser.parse(%w(--skip-ci-status)) }.to raise_error(RuntimeError)
+  end
+  
+  it 'should handle --skip-ci-status and build correctly' do
+    Moonshot.config = Moonshot::ControllerConfig.new
+    Moonshot.config do |c|
+      c.build_mechanism = Moonshot::BuildMechanism::GithubRelease.new('')
+    end
+    cli_dispatcher = Moonshot::CommandLineDispatcher.new('build', subject, {})
```

Comment:
The difference between these two test cases is not clear from the description.

The one with non-standard config should have a clearer description or use a context to do that.